test/exercises/Mandelbrot/solutions/mandelbrot4-taskpar.chpl

```diff
@@ -17,7 +17,7 @@ config const maxSteps = 50;
 // 
 // The number of tasks we should spawn.
 //
-config const numTasks = here.numCores; // Cores on locale #0.
+config const numTasks = here.numPUs(physical=true); // Cores on locale #0.
```

Comment:
Any reason you don't just do `numPUs()` everywhere since physical is true by default?